WordPress/src/main/java/org/wordpress/android/datasets/PeopleTable.java

```diff
@@ -26,6 +25,7 @@ private static SQLiteDatabase getWritableDb() {
     public static void createTables(SQLiteDatabase db) {
         db.execSQL("CREATE TABLE " + PEOPLE_TABLE + " ("
                 + "person_id               INTEGER DEFAULT 0,"
+                + "site_id                 TEXT,"
```

Comment:
> I feel like site_id here makes more sense just because the only use of it is for the network calls. 

If it's used for network calls, it's not "local". Local_id was used for blogs/sites because we can have a mixture of self hosted and wordpress.com sites, so we can't use the id provided by a network response.

Will this be used for wpcom sites only?